trie/triedb/pathdb/database.go

```diff
@@ -410,15 +419,18 @@ func (db *Database) Initialized(genesisRoot common.Hash) bool {
 
 // SetBufferSize sets the node buffer size to the provided value(in bytes).
 func (db *Database) SetBufferSize(size int) error {
```

Comment:
the func name is `SetBufferSize` ,but now internal logic is gone,  mismatch now
may be we can keep SetBufferSize logic, but not to call it outside
like in func  `func (h *handler) enableSyncedFeatures()`

Reply:
Yes, fixed